### ISSUE TITLE
Published version of koa-proxy apparently has a .git/index file which breaks subsequent npm install

### ISSUE THREAD

**Issue by usergenic** [edorivai/koa-proxy#7]
I think you published a .git/index file with koa-proxy and it causes this error on subsequent attempts to use npm install command for any reason.
```shell
λ npm i
npm ERR! path /something/something/node_modules/koa-proxy
npm ERR! code EISGIT
npm ERR! git /something/something/node_modules/koa-proxy: Appears to be a git repo or submodule.
npm ERR! git     /something/something/node_modules/koa-proxy
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/usergenic/.npm/_logs/2019-07-15T19_49_20_835Z-debug.log
```

**Comment by nancy-Gao** [comment]
I have the same problem for a long time, it's necessary for me to fix this. When can fix it?

**Comment by LarsDenBakker** [comment]
Having the same issues

**Comment by usergenic** [comment]
Hey guys, just a small FYI, via https://twitter.com/EdoRivai/status/1151265569011884033 @edorivai is on vacation until July 22. Hang tight :)

**Comment by LarsDenBakker** [comment]
Meanwhile I've been using v1.0.0-alpha.0 which doesn't have the git folder.

**Comment by edorivai** [comment]
Should be fixed in `koa-proxy@1.0.0-alpha.3`

**Comment by usergenic** [comment]
Thanks, @edorivai !
